packages/components/src/components/Trigger/Trigger.js

```diff
@@ -37,120 +38,229 @@ const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
+          <span className="resourcekind">
+            {intl.formatMessage({
+              id: 'dashboard.triggerDetails.triggerTemplate',
+              defaultMessage: 'TriggerTemplate:'
+            })}
+          </span>
 
           <Link
+            className="triggerresourcelink"
             to={urls.triggerTemplates.byName({
               namespace: eventListenerNamespace,
               triggerTemplateName: trigger.template.name
             })}
           >
-            <span className="linkicon">
-              <LinkIcon />
-            </span>
-            <span className="truncate-text-end" title={trigger.template.name}>
-              {trigger.template.name}
-            </span>
+            <span title={trigger.template.name}>{trigger.template.name}</span>
           </Link>
         </div>
+      </div>
 
-        <div className="triggerinfo">
-          <h3>
-            {intl.formatMessage({
-              id: 'dashboard.parameters.title',
-              defaultMessage: 'Parameters'
-            })}
-          </h3>
-
-          <Table
-            className="parameterstable"
-            headers={tableHeaders}
-            rows={triggerParams}
-            emptyTextAllNamespaces={intl.formatMessage({
-              id: 'dashboard.trigger.noParams',
-              defaultMessage: 'No parameters found for this Trigger.'
-            })}
-            emptyTextSelectedNamespace={intl.formatMessage({
-              id: 'dashboard.trigger.noParams',
-              defaultMessage: 'No parameters found for this Trigger.'
-            })}
-          />
-          {trigger.interceptor && (
-            <>
-              <div className="interceptor">
-                <h3>
-                  {intl.formatMessage({
-                    id: 'dashboard.triggerDetails.interceptorName',
-                    defaultMessage: 'Interceptor: '
-                  })}
-                  {trigger.interceptor.objectRef.name}
-                </h3>
-                <h3>
-                  {intl.formatMessage({
-                    id: 'dashboard.triggerDetails.interceptorHeaders',
-                    defaultMessage: 'Headers'
-                  })}
-                </h3>
-              </div>
-              <Table
-                headers={tableHeaders}
-                rows={interceptorValues}
-                emptyTextAllNamespaces={intl.formatMessage({
-                  id: 'dashboard.trigger.noHeaders',
-                  defaultMessage: 'No headers found for this interceptor.'
-                })}
-                emptyTextSelectedNamespace={intl.formatMessage({
-                  id: 'dashboard.trigger.noHeaders',
-                  defaultMessage: 'No headers found for this interceptor.'
-                })}
-                isSortable={false}
-              />
-            </>
-          )}
-        </div>
+      <div className="trigger--interceptors">
+        {trigger.interceptors && trigger.interceptors.length !== 0 && (
```

Comment:
do we have / need an empty state for this?

Reply:
I don't think we need an empty state, since a Trigger does not need to have any Interceptors